src/global/interfaces/link-button-properties.ts

```diff
@@ -84,7 +110,13 @@ export interface ButtonProperties<T = any> extends AccessibilityProperties {
  */
 export interface LinkButtonProperties<ParameterType = any>
   extends LinkProperties,
-    ButtonProperties<ParameterType> {}
+    ButtonProperties<ParameterType> {
+  /**
+   * Used to set the correct value of LinkButtonRenderVariables under certain conditions.
+   * E.g. if href is not set, use getLinkRenderVariables(...), otherwise use getButtonRenderVariables(...).
+   */
+  resolveRenderVariables: () => LinkButtonRenderVariables;
```

Comment:
I think this can be omitted.
Instead it might be easier to provide a function `getLinkButtonRenderVariables` (and maybe additionally `getStaticLinkButtonRenderVariables`) to simplify the usage.
@jeripeierSBB, @DavideMininni-Fincons Thoughts?

Reply:
Sounds good :)

Reply:
I think we are able to ensure this via our code review process and also having the reference implementations. I would prefer to avoid having the method in the interface.

Reply:
It's fine for me; method removed